go/libraries/doltcore/table/editor/index_editor_test.go

```diff
@@ -52,8 +52,9 @@ var id2, _ = uuid.NewRandom()
 var id3, _ = uuid.NewRandom()
 
 func TestIndexEditorConcurrency(t *testing.T) {
-	format := types.Format_Default
+	format := types.Format_LD_1
```

Comment:
Probably better to skip?

Reply:
these tests are specific to `__LD_1__` and I think they're useful to that end